services/commands-service.ts

```diff
@@ -104,6 +105,11 @@ export class CommandsService implements ICommandsService {
 
 				this.$errors.fail("Unable to execute command '%s'. Use '$ %s %s --help' for help.", beautifiedName, this.$staticConfig.CLIENT_NAME.toLowerCase(), beautifiedName);
 				return false;
+			} else {
+				if(_.any(this.$commandsServiceProvider.allDynamicCommands())) {
```

Comment:
if you use my code, what will happen in case there aren't any dynamic commands :cat: